canonical/ght
# WD-6437 - Remove some APAC regions from replication

## Done
Removed the 6 APAC regions from the defaults

## QA
Replicate a test role in `apac` and see there is no:
```
Chengdu
Hangzhou
Nanjing
Shenzhen
Wuhan
Xiamen
```

Fixes https://warthogs.atlassian.net/browse/WD-6437
